packages/api/tests/lib/writeRecords/test-write-execution.js

```diff
@@ -468,16 +406,14 @@ test.serial('writeExecutionRecordFromMessage() does not persist records to Dynam
     writeExecutionRecordFromMessage({ cumulusMessage, knex }),
     { message: 'execution RDS error' }
   );
-  t.false(await executionModel.exists({ arn: executionArn }));
   t.false(await executionPgModel.exists(knex, { arn: executionArn }));
   t.false(await t.context.esExecutionsClient.exists(executionArn));
 });
 
-test.serial('writeExecutionRecordFromMessage() does not persist records to Dynamo/RDS/Elasticsearch if Elasticsearch write fails', async (t) => {
+test.serial('writeExecutionRecordFromMessage() does not persist records to RDS/Elasticsearch if Elasticsearch write fails', async (t) => {
```

Comment:
Super Nit (and not this PR) `does not persist records in RDS`

Reply:
Semantics, and disagree with `in` over `to`, as the former to my mind suggests _existing_ records will no longer be persisted. 🙂

Reply:
To be clear the nit was really more that Elasticsearch in `RDS/Elasticsearch` is redundant.